editing_files.md

```diff
@@ -0,0 +1,17 @@
+## Editing Files
+
+On Unix Systems, there are three main families of editors: `ed`, `vi` and `emacs`. The `ed` editor was the first of these three editors created. It is a line-oriented editor instead of a full-screen editor and will feel very primitive compared to `vi` and `emacs`. It is can be useful on the rare occasion when one is dealing with a console terminal. (I wishe no one ever to have to suffer this) If you run:
```

Comment:
vi was a descendent of ed, I thought? Not sure about the utility of describing these as "families".